libraries/src/Toolbar/Toolbar.php

```diff
@@ -159,8 +159,8 @@ public function setFactory(ToolbarFactoryInterface $factory): self
 	/**
 	 * Append a button to toolbar.
 	 *
-	 * @param   ToolbarButton  $button  The button instance.
-	 * @param   array          $args    The more arguments.
+	 * @param   ToolbarButton|string  $button  The button instance.
```

Comment:
```suggestion
	 * @param   ToolbarButton|string  $button  The button instance. String type is deprecated and will be removed with Joomla 5.0
```

I thought about something like this.

Reply:
then it should be a proper deprecated note